integration-tests/mock/pkg/http/whiteflag/init.go

```diff
@@ -0,0 +1,245 @@
+	"github.com/iotaledger/iota.go/merkle"
+	"github.com/iotaledger/iota.go/pow"
+	"github.com/iotaledger/iota.go/transaction"
+	"github.com/iotaledger/iota.go/trinary"
+)
+
+var (
+	powFunc pow.ProofOfWorkFunc
+	data    *whiteFlagData
+)
+
+type whiteFlagData struct {
+	latestMilestoneHash  trinary.Hash
+	latestMilestoneIndex uint32
+	coordinatorAddress   trinary.Hash
+
+	milestones []whiteFlagMilestone
+}
+
+type whiteFlagMilestone struct {
+	milestoneBundle          []trinary.Trytes
+	includedMigrationBundles [][]trinary.Trytes
+}
+
+func init() {
+	cfg := config.GetConfig()
+
+	_, powFunc = pow.GetFastestProofOfWorkImpl()
+
+	log.Println("creating migration bundles...")
+	includedBundles, err := createIncludedBundles(cfg.WhiteFlag)
+	if err != nil {
+		log.Fatalf("failed to create bundles: %s", err)
+	}
+	log.Printf("created bundles for %d milestone indices\n", len(includedBundles))
+
+	log.Println("creating milestones...")
+	data, err = createMilestones(cfg.Coordinator, includedBundles)
+	if err != nil {
+		log.Fatalf("failed to create milestones: %s", err)
+	}
+	log.Printf("mocked coordinator: {address=%s, depth=%d, MWM=%d, latestMSIndex=%d}\n",
+		data.coordinatorAddress, cfg.Coordinator.TreeDepth, cfg.Coordinator.MWM, data.latestMilestoneIndex)
+
+	// register the API commands
+	httpapi.RegisterHandler(strings.ToLower(GetNodeInfoCommand), getNodeInfo)
+	httpapi.RegisterHandler(strings.ToLower(GetWhiteFlagConfirmationCommand), getWhiteFlagConfirmation)
+
+	log.Println("white flag API initialized")
+}
+
+func createIncludedBundles(cfg config.WhiteFlagConfig) (map[uint32][][]trinary.Trytes, error) {
+	iotaAPI := new(legacyapi.API)
+
+	includedBundles := make(map[uint32][][]trinary.Trytes)
+	for msIndex, migrations := range cfg.Migrations {
+		var bundles [][]trinary.Trytes
+		for _, migration := range migrations {
+			addr, err := address.GenerateAddress(cfg.Seed, migration.Index, migration.Security, true)
+			if err != nil {
+				return nil, fmt.Errorf("failed to generate address: %w", err)
+			}
+			inputs := []legacyapi.Input{{
+				Balance:  migration.Balance,
+				Address:  addr,
+				KeyIndex: migration.Index,
+				Security: migration.Security,
+			}}
+			migrationAddress, err := generateMigraionAddress(migration.Ed25519Address)
+			if err != nil {
+				return nil, fmt.Errorf("failed to generate migration address from config: %w", err)
+			}
+			transfers := []bundle.Transfer{{
+				Address: migrationAddress,
+				Value:   migration.Balance,
+			}}
+
+			bndl, err := iotaAPI.PrepareTransfers(cfg.Seed, transfers, legacyapi.PrepareTransfersOptions{
+				Inputs: inputs,
+			})
+			if err != nil {
+				return nil, fmt.Errorf("failed to create bundle: %w", err)
+			}
+			// PrepareTransfers returns the transaction in the reversed order, so we must reverse again
+			for i, j := 0, len(bndl)-1; i < j; i, j = i+1, j-1 {
+				bndl[i], bndl[j] = bndl[j], bndl[i]
+			}
+			bundles = append(bundles, bndl)
```

Comment:
I've noticed that we don't PoW the "migration bundles", do you think it makes sense to still do that?

Reply:
I guess, it does not hurt to PoW them, if that is not relevant for the testcase one can set the MWM to 0 anyways.